middleware.js

```diff
@@ -265,18 +269,18 @@ module.exports = function(options) {
 
     // Compare mtimes
     fs.stat(sassPath, function(err, sassStats) {
-      if (err) {
-        error(err);
```

Comment:
If sassPath doesn't exist then consumer of sass middleware must have wrong input path provider. Error/fail fast is the current behavior, this is intentional?